### PR TITLE
Resize mounts by default

### PR DESCRIPTION
**Pull request by feikesteenbergen**
Volumes that should not be erased_on_boot may have been created
from different snapshots. One of the reasons to do this is to be able
to scale data stores (PostgreSQL, Cassandra) once they fill up.
What happens in this case is that the volume is resized, however the
filesystem still has the previous size.
Resizing the filesystem to the volume seems like a win, we can do
it unconditionally as resize2fs will figure out by itself whether
a resize is needed.

Addresses issue #252

### CODE REVIEW

File: runtime/opt/taupage/init.d/10-prepare-disks.py [hunk 1]
```diff
@@ -125,6 +125,9 @@ def format():
         def mount():
             mount_partition(partition, mountpoint, options, filesystem, os.path.isdir(mountpoint), already_mounted)
 
+        def resize():
+            subprocess.check_call(['resize2fs', partition])
```
Comment (slitsche) [inline runtime/opt/taupage/init.d/10-prepare-disks.py]: What if it the file system is not ext4? Shouldn't we call `xfs_growfs` for xfs?

Comment (feikesteenbergen) [inline runtime/opt/taupage/init.d/10-prepare-disks.py]: Yes, we need that

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**